packages/frontend/src/stdlib/theories.tsx

```diff
@@ -262,6 +262,66 @@ stdTheories.add(
     },
 );
 
+stdTheories.add(
+	{
+		id: "nullable-causal-loop",
+		name: "Nullable Causal Loop",
+	},
+	(meta) => {
+		const thNullableSignedCategory = new catlog.ThNullableSignedCategory();
+	    return new Theory({
+			    ...meta,
+			    theory: thNullableSignedCategory.theory(),
+			    onlyFreeModels: true,
+			    types: [
+				    {
+				        tag: "ObType",
+					      obType: { tag: "Basic", content: "Object" },
+					      name: "Type",
+					      description: "Type or class of things",
```

Comment:
For consistency, all of these names and descriptions should mirror those for ordinary causal loop diagrams where appropriate. E.g., this first name should be "Variable".